pkg/cmd/roachtest/tests/restore.go

```diff
@@ -612,6 +565,17 @@ func registerRestore(r registry.Registry) {
 						t.L().Printf("pausing RESTORE job")
 						// Pause the job and wait for it to transition to a paused state.
 						_, err = conn.ExecContext(ctx, `PAUSE JOB $1`, jobID)
+						if err != nil {
```

Comment:
I think we should error out if we're not able to pause atleast once? Otherwise this test is not doing what it says it is, wdyt

Reply:
good idea. Will send the patch to master and then add it to the backports.

Reply:
opened this on master https://github.com/cockroachdb/cockroach/pull/91024

Reply:
added cherry pick to this branch